tensorflow/contrib/tensorrt/python/trt_convert.py

```diff
@@ -69,7 +69,8 @@ def tensorrt_rewriter_config(max_batch_size=1,
                              minimum_segment_size=3,
                              is_dynamic_op=False,
                              maximum_cached_engines=1,
-                             cached_engine_batch_sizes=None):
+                             cached_engine_batch_sizes=None,
+                             use_calibration=True):
```

Comment:
I feel not_use_calibration and precision_mode are mutually exclusive. When using calibration, you definitely need precision_mode otherwise you wouldn't figure that out from the graph. When not using calibration, you don't need precision_mode because your graph specifies what precision it needs using QuantizeAndDequantize ops (QuantizeAndDequantize has a num_bits flag). In fact, a user could even do mixed-precision execution in *one* graph by adding some QuantizeAndDequantizes with num_bits=8 and some other QuantizeAndDequantizes with num_bits=4.

Reply:
Thanks, that is a good point. Currently all the TRT engines are using same precision mode and we may want to change that (maybe we can add something to the custom backend API design to allow different settings for different subgraphs).

Reply:
So I propose not adding INT8_NO_CALIBRATION. precision=int8 and use_calibration=false trigger dynamic range collection and quantization. WDYT, Guangda?

Reply:
@aaroey

Reply:
Agreed. Closing.